app/src/main/java/org/mozilla/fenix/onboarding/SyncCFRPresenter.kt

```diff
@@ -51,12 +52,15 @@ class SyncCFRPresenter(
                         false -> Onboarding.syncCfrImplicitDismissal.record(NoExtras())
                     }
                 }
-            ) {
-            }.apply {
+            ).apply {
                 syncCFR = this
                 show()
-                Onboarding.synCfrShown.record(NoExtras())
             }
+
+            context.settings().showSyncCFR = false
+            context.settings().shouldShowJumpBackInCFR = false
```

Comment:
Why should `shouldShowJumpBackInCFR` be false here?

Reply:
This is part of the acceptance criteria https://github.com/mozilla-mobile/fenix/issues/26756#issue-1358275594
Jump back in “Should not be shown if the synced tab pickup message is already showing or if the user has already seen the synced tab pickup message”